Sources/StreamChatSwiftUI/ChatChannel/Composer/TrailingComposerView.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+public struct TrailingComposerView: View {
+    
+    @Injected(\.utils) private var utils
+        
+    @EnvironmentObject var viewModel: MessageComposerViewModel
```

Comment:
Why not use our `Injected` instead of `Environment`?

Reply:
We don't have the VMs as `Injected` values. In this case, we just need it in the child view, so it's ok to pass it like this.